src/ipolates.f90

```diff
@@ -46,17 +68,23 @@ subroutine ipolates_grid(ip, ipopt, grid_in, grid_out, mi, mo, km, ibi, li, gi,
 
     select case(ip)
     case(BILINEAR_INTERP_ID)
-       CALL interpolate_bilinear(IPOPT,grid_in,grid_out,MI,MO,KM,IBI,LI,GI,NO,RLAT,RLON,IBO,LO,GO,IRET)
+       CALL interpolate_bilinear(IPOPT,grid_in,grid_out,MI,MO,KM,IBI&
+            &,LI,GI,NO,RLAT,RLON,IBO,LO,GO,IRET)
     case(BICUBIC_INTERP_ID)
-       CALL interpolate_bicubic(IPOPT,grid_in,grid_out,MI,MO,KM,IBI,LI,GI,NO,RLAT,RLON,IBO,LO,GO,IRET)
+       CALL interpolate_bicubic(IPOPT,grid_in,grid_out,MI,MO,KM,IBI&
```

Comment:
For easy review, what I suggest is to do all whitespace only reformatting as a separate PR, with no code changes. So merge this PR, then reformat the code however you like, and then do a PR with that.

That way we can easily scan the whitespace changes to ensure no code was accidently changed.